src/proto/thread_telemetry.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+option optimize_for = LITE_RUNTIME;
+
+package threadnetwork;
+
+// Thread Telemetry data definition.
+// The field range for your data definition is determined as:
+// ---------------------------------------------------------------------------
+// | Field Range  | Logging From
+// | [1 - 500)    | Primary fields logged from OTBR-agent/OpenThread.
+// | [500-600)    | OTBR vendor fields logged from OTBR-agent/OpenThread.
+// | Other        | Reserved for now.
+// ---------------------------------------------------------------------------
+// Usage:
+// Delete metrics: do not directly delete metrics. Deprecate it instead.
```

Comment:
`metrics` seems not defined in the context. Maybe just refer to it by `field`?

Reply:
Done.